vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -946,10 +948,140 @@ public void failure(VimeoError error) {
     // </editor-fold>
 
     // -----------------------------------------------------------------------------------------------------
-    // Editing (Video, User)
+    // Editing (Video, User, Album)
     // -----------------------------------------------------------------------------------------------------
     // <editor-fold desc="Editing (Video, User)">
 
+    /**
+     * Call this method to create a new album.
+     *
+     * @param name         A non-blank string that will be the new name of the album.
+     * @param albumPrivacy A representation of the viewing privacy for the album. The expectation is that if the
+     *                     album is password protected that this object will also contain the required password.
+     * @param description  A nullable description for the album.
+     * @param parameters   Additional less common parameters that modify the album.
+     * @param callback     A callback that will receive the results of the network request.
+     * @return A Call object or null if a client-side initialization error has occurred. In the event of an error,
+     * the callback will still be notified.
+     */
+    @Nullable
+    public Call<Album> createAlbum(@NotNull final String name,
+                                   @NotNull final AlbumPrivacy albumPrivacy,
+                                   @Nullable final String description,
+                                   @Nullable final Map<String, Object> parameters,
+                                   @NotNull final VimeoCallback<Album> callback) {
+        if (VimeoNetworkUtil.validateString(name, "Name can't be empty in createAlbum.", callback)) {
+            final Map<String, Object> validParameters =
+                    VimeoNetworkUtil.prepareAlbumEditParameters(name, albumPrivacy, description, parameters, callback);
```

Comment:
The `prepareAlbumEditParameters` has multiple responsibilities. It is creating a map but it is also validating whether the client provided a password. The validation could be moved outside of the method and the method could be renamed to `createAlbumEditParamsMap` to signify what it is doing.